tests/bool.rs

```diff
@@ -191,7 +191,7 @@ mod test {
         let egraph = Scheduler::Compress(Limits::default()).run(&atoms5.to_egraph(), &all_rules);
         candidates = Ruleset::cvec_match(&egraph);
         let rules5 = candidates.minimize(all_rules.clone(), Limits::default());
-        assert_eq!(rules5.len(), 16);
+        assert_eq!(rules5.len(), 14);
```

Comment:
Wait so this works better than the old minimizer? Why?

Reply:
not 100% sure, but the 14 rules and 16 rules are equi-derivable, and i'm not sure these tests that assert about the numbers of rules are useful anyways so i didn't think too much about it